scripts/setup

```diff
@@ -483,6 +483,14 @@ then
     diff --suppress-common-lines --side-by-side tmp/mapping_before_setup.txt tmp/mapping_after_setup.txt
 fi
 
+info_msg "- Load Virtua CREATE operation logs: ${DATA_PATH}/operation_logs_creations.json"
+eval ${PREFIX} invenio fixtures migrate_virtua_operation_logs -f ${DATA_PATH}/operation_logs_creations.json
+info_msg "- Load Virtua UPDATE operation logs: ${DATA_PATH}/operation_logs_updates.json"
+eval ${PREFIX} invenio fixtures migrate_virtua_operation_logs -f ${DATA_PATH}/operation_logs_updates.json
+
```

Comment:
Why are the update and creation operation logs in separated files?

Reply:
In plexus, you have now one file to load `operation_logs.json`